docs/HazardAnalysis/HazardAnalysis.tex

```diff
@@ -158,10 +158,16 @@ \subsubsection*{Web Client}
 
 \section{Critical Assumptions}
 
-\wss{These assumptions that are made about the software or system.  You should
-minimize the number of assumptions that remove potential hazards.  For instance,
-you could assume a part will never fail, but it is generally better to include
-this potential failure mode.}
+\begin{itemize}
+    \item The Energy Measurement Model will provide accurate and consistent energy consumption metrics across different platforms (Windows, macOS, Linux). There are no discrepancies in measurements due to platform differences that could result in ineffective refactoring.
+    \item Refactoring will not introduce new bugs or alter the intended functionality of the original Python code.
```

Comment:
I don't know if this assumption is right to make. The comment the prof left on the doc mentioned that we shouldn't make assumptions that negate risks. Not to mention we have a testing module meaning we clearly assume that there _could_ be bugs

Reply:
I agree, going to remove this assumption